dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -154,6 +177,35 @@ private static void cleanStatePrivate2(DropwizardTestSupport<DockstoreWebservice
         application.run("db", "migrate", configPath, "--include", "1.5.0");
     }
 
+    /**
+     * Drops and recreates database from migrations for test confidential 2 without VerificationInformation Migration
+     * @param support reference to testing instance of the dockstore web service
+     * @throws Exception
+     */
+    public static void cleanStatePrivate2NoVerificationMigration(DropwizardTestSupport<DockstoreWebserviceConfiguration> support) throws Exception {
+        Application<DockstoreWebserviceConfiguration> application;
+        application= support.getApplication();
+        application.run("db", "drop-all", "--confirm-delete-everything", CONFIG_PATH);
+        List<String> migrationList = Arrays.asList("1.3.0.generated", "1.3.1.consistency", "test.confidential2", "1.4.0");
+        runMigration(migrationList, application, CONFIG_PATH);
+        application.run("db", "migrate", CONFIG_PATH, "--include", "1.5.0", "-c", "8");
```

Comment:
Don't use count (will break easily if new migrations inserted), just run all of 1.5.0.